core/regions/src/main/java/com/kape/vpnregions/utils/RegionListProvider.kt

```diff
@@ -26,7 +26,12 @@ class RegionListProvider(
     fun getOptimalServer(): VpnServer {
         val optimalServerOptions = _servers.value.filter { it.autoRegion && it.isGeo.not() }
         return if (optimalServerOptions.none { it.latency != null }) {
-            optimalServerOptions.random()
```

Comment:
I agree. I'm working based on assumptions here but the only time they might be empty is if a garbage collector kicked in and wiped them. So, re-initiating to the default list sounded like a good start. Any thoughts/ideas are welcome. 
Just to note I have never been able to reproduce this particular crash.

Reply:
yeah, I'm thinking of any other potential crashes triggered by this empty case on other parts of the logic, and whether we should officially handle the possibility of this being empty in that case... Let's go ahead with just covering this particular use case given the stack trace, if we get crashes elsewhere then we focus on fixing the server list being empty scenario, rather than the new specific crash.